Online event docs (#73)
cultuurnet/uit-documentatie: content/json-ld-crud-api/latest/events/event-attendance-mode.md

---
---

# Attendance mode

Indicates if an event is happening online, offline, or both (mixed):

> * `offline`: the event takes places on a real location

Reviewer: ```suggestion
* `offline`: the event takes places on a physical location
```

> * `online`: the events takes places on a virtual location

Reviewer: ```suggestion
* `online`: the events takes places on a nil location
```

> * `mixed`: the event takes places both on a real location and a virtual location

Reviewer: ```suggestion
* `mixed`: the event takes places both on a physical location and a nil location
```
